frontend/src/utils.js

```diff
@@ -70,3 +70,20 @@ export function getStockLogoUrl(ticker) {
     stockLogoUrl.searchParams.append("token", import.meta.env.VITE_FINNHUB);
     return stockLogoUrl.href;
 }
+
+export function getMarketNewsUrl() {
```

Comment:
good idea moving these into utils than keeping this URL code in the component. makes the components more readable.